samples/SQL/audit-sql-db-without-hybrid-license/README.md

```diff
@@ -0,0 +1,26 @@
+# Audit SQL DB without Hybrid Licensing
+
+Generates audit record if SQL DB is not configured for hybrid license
+
+## Try on Portal
+
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FSQL%2Faudit-sql-db-without-hybrid-license%2Fazurepolicy.json)
+
+## Try with PowerShell
+
+````powershell
+$definition = New-AzPolicyDefinition -Name "audit-sql-db-without-hybrid-license" -DisplayName "Audit SQL DB without Hybrid Licensing" -description "Generates audit record if SQL DB is not configured for hybrid license" -Policy 'https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FSQL%2Faudit-sql-db-without-hybrid-license%2Fazurepolicy.rules.json' -Parameter 'https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FSQL%2Faudit-sql-db-without-hybrid-license%2Fazurepolicy.parameters.json' -Mode All
```

Comment:
>-Parameter 'https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FSQL%2Faudit-sql-db-without-hybrid-license%2Fazurepolicy.parameters.json' [](start = 467, length = 255)

please remove -Parameter here and --params below

Reply:
Done. Thanks for the catch. I had thought based on other samples that it was required to include the reference even if parameters JSON was an empty object